Dockerfile

```diff
@@ -0,0 +1,27 @@
+# This Dockerfile is a temporary solution so that we can test our centos
```

Comment:
It would be nice to hide these intermediate Dockerfiles under `.kitchen/` dir.
Just to avoid confusion from users when we place `Dockerfile` in the root dir of repo.